test/base/test_batch.py

```diff
@@ -8,6 +8,7 @@
 
 
 def test_batch():
+    assert list(Batch()) == []
```

Comment:
@duburcqa in #119 you can change it as you want. I also agree with you that `__iter__` is not a good solution, but I approved it.

Reply:
Will do! :smile: